modules/reindex/src/main/java/org/elasticsearch/index/reindex/RemoteScrollableHitSource.java

```diff
@@ -150,25 +147,48 @@ HttpPost buildStartRequest() {
                 }
             }
             if (useScan) {
-                uri.append("&search_type=scan");
+                uri.addParameter("search_type", "scan");
```

Comment:
Yeah - this only uses it if sorting by _doc isn't available.

Officially I'm not sure exactly what version of Elasticsearch it is ok to say that reindex-from-remote doesn't support, but I am sure it needs to support 1.x. So scan it is.